### PR TITLE
Redraft soap steps

### PR DESCRIPTION
**Pull request by SueSmith**
@loopDelicious had a go at redrafting the soap doc, can hold off until your blog post is up and add it in the right sidebar, added new pics too in case any of them are handy..

### CODE REVIEW

File: src/pages/docs/postman/sending-api-requests/making-soap-requests.md [hunk 1]
```diff
@@ -23,15 +23,53 @@ warning: false
 
 ---
 
-Many people think of Postman as an advanced REST client. Beyond REST, Postman is a tool that handles any calls sent over HTTP. This means that you can use Postman to interact with protocol-agnostic APIs - such as SOAP and [GraphQL](/docs/postman/sending-api-requests/graphql/), which can both utilize HTTP, just like REST.
+Although Postman is primarily a REST client, you can make other types of HTTP call, including to protocol-agnostic services such as SOAP and [GraphQL](/docs/postman/sending-api-requests/graphql/).
```
Comment (loopDelicious) [inline src/pages/docs/postman/sending-api-requests/making-soap-requests.md:26]: Re: positioning: I'd prefer that WE don't call it a REST client, but acknowledge that it's known as a REST client or frequently used as a REST client.

**Comment by SueSmith** [comment]
@loopDelicious added header content type section 🤦‍♀️ / adjusted rest client wording / updated template - will get the blog bit updated when it's live!

**Comment by loopDelicious** [comment]
@SueSmith https://blog.postman.com/making-http-soap-requests-with-postman/

**Comment by SueSmith** [comment]
@loopDelicious blog link updated - if you can approve this for me I'll get it merged!

**Comment by SueSmith** [comment]
@loopDelicious yes for sure, I would like to do _a lot_ more to highlight templates

**Pull request merged**